elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -2,193 +2,89 @@
 
 $LOCAL: tab;
 
-/// ===========================================================================
-/// Component Specific SASS Vars
-/// ===========================================================================
+// Active tab styles
 
-$pfe-tabs__tab--PaddingRight-factor:  3.375; // 54px at 16px base.
-$pfe-tabs__tab--PaddingBottom-factor:  1.5; // 54px at 16px base.
+:host(:focus),
+:host(:focus-visible) {
+  outline: #{pfe-var(ui--focus-outline-width)} #{pfe-var(ui--focus-outline-style)} var(--pfe-tabs--focus);
+}
 
 /// ===========================================================================
-/// DEFAULT TAB
+/// HORIZONTAL TAB ORIENTATION
 /// ===========================================================================
 
 :host {
-  --pfe-tabs--main:         transparent;
-  --pfe-tabs--aux:          #{pfe-color(surface--lightest--text)};
-  --pfe-tabs--link:         #{pfe-color(surface--lightest--link)};
-  --pfe-tabs--focus:        #{pfe-color(surface--lightest--link--focus)};
-
-  position: relative;
-  display: block;
-  margin: 0 0 -1px;
-
-  // Padding
-  padding-top:    #{pfe-var(container-padding)};
-  padding-right:  calc(#{pfe-var(container-padding)} * #{$pfe-tabs__tab--PaddingRight-factor});;
-  padding-bottom: calc(#{pfe-var(container-padding)} * #{$pfe-tabs__tab--PaddingBottom-factor});
-  padding-left:   #{pfe-var(container-padding)};
-  // Border
-  border: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} transparent;
-  border-bottom: 0;
-  // Inner style
-  background-color: var(--pfe-tabs--main);
-  color: var(--pfe-tabs--aux);
-  text-transform: var(--pfe-tabs__tab--TextTransform, none); // exposing this for RH
-  font-weight: #{pfe-var(font-weight--normal)};
-
-  white-space: nowrap;
-  cursor: pointer;
+  @include pfe-tab;
 }
 
 :host([aria-selected="true"]) {
-  --pfe-tabs--main: #{pfe-color(surface--lightest)};
-
-  // Border
-  border-color: #{pfe-color(surface--border)};
-  border-bottom: 0;
+  @include pfe-tab-selected;
```

Comment:
@chrisdo1 What was the goal in abstracting all the styles into mixins for this component?

Reply:
@castastrophe I have moved all of the tab mixin code back into their respective scss files (except for one).